docker/bragi/Dockerfile

```diff
@@ -39,17 +39,14 @@ RUN --mount=type=cache,target=/home/mimirsbrunn/target \
 # Target image
 # ============
 
-FROM debian:buster-slim
+FROM rust:1.60-alpine3.15
 
 WORKDIR /srv
 
 ENV DEBIAN_FRONTEND noninteractive
 
-RUN apt-get update
 # curl required for healthchecks
-RUN apt-get install -y curl libcurl4 sqlite3 
-RUN apt-get clean
-RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+RUN apk --update --no-cache add curl sqlite bash 
```

Comment:
Why do we need *bash*? Is this a general good practice to be able to open a CLI on a running container?

Reply:
Oh yeah, this is a bash script: https://github.com/Qwant/mimirsbrunn/blob/master/docker/run_with_default_config.sh :+1:

Reply:
no the goal is to reduce the vulnerabilities and switch to alpine drastically improve this problem
no mimirsbrun is runned via jobs so is less vulnerable than bragi that is a standalone app
yes I've realized that a binary builded in debian can't be executed in alpine my bad ()